src/main/java/seedu/address/logic/parser/UnpoolCommandParser.java

```diff
@@ -1,45 +1,29 @@
 package seedu.address.logic.parser;
 
 import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
-import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
-import static seedu.address.logic.parser.CliSyntax.PREFIX_PHONE;
-
-import java.util.stream.Stream;
 
+import seedu.address.commons.core.index.Index;
 import seedu.address.logic.commands.UnpoolCommand;
 import seedu.address.logic.parser.exceptions.ParseException;
-import seedu.address.model.person.Name;
-import seedu.address.model.person.Phone;
-import seedu.address.model.person.driver.Driver;
 
+/**
+ * Parses input arguments and creates a new DeleteCommand object
+ */
 public class UnpoolCommandParser implements Parser<UnpoolCommand> {
+
     /**
-     * Parses the given {@code String} of arguments in the context of the UnpoolCommand
-     * and returns an PoolCommand object for execution.
+     * Parses the given {@code String} of arguments in the context of the DeleteCommand
```

Comment:
Should this be DeleteCommand instead of PoolCommand?

Reply:
thanks, i have rectified this issue.